.github/workflows/main.yml

```diff
@@ -25,6 +25,10 @@ jobs:
           - stable
           - 1.83.0
     name: test
+    env:
+      PKCS11_MODULE: /usr/lib/softhsm/libsofthsm2.so
+      SOFTHSM2_CONF: /tmp/softhsm2.conf
+      RUSTFLAGS: --cfg test_hsm
```

Comment:
Hmm, would it be possible to separate these config settings (and the Install SoftHSM step) to a separate test job in this file? That way, in case something goes wrong with these new steps, we can more easily distinguish from the regular (existing) tests.

Reply:
Yep, that separation makes sense, but indeed I was worried about the installation steps themselves breaking.

I suppose we can keep them here for now and if they become problematic in the future, we can separate them out.